radiometric_normalization/pif.py

```diff
@@ -17,12 +17,12 @@
 import numpy
 from collections import namedtuple
 
-from radiometric_normalization import pca_filter
-from radiometric_normalization import robust
-from radiometric_normalization import filtering
+from radiometric_normalization.radiometric_normalization import pca_filter
```

Comment:
The relative import wasn't working?

Reply:
Yeah sorry. It was working. here is the fix 7ba51d46bd9366a0d70e6930bb95b060222e1234